pkg/controllers/management/node/controller.go

```diff
@@ -432,6 +435,14 @@ func validateCustomHost(obj *v3.Node) error {
 	}
 
 	customConfig := obj.Spec.CustomConfig
+
+	//validate node hostname when creation
+	hostname := obj.Spec.RequestedHostname
```

Comment:
this field is nullable. The validation should only be done when the value is not nil